paasta_tools/cli/cmds/logs.py

```diff
@@ -291,7 +297,7 @@ def paasta_log_line_passes_filter(
     if not check_timestamp_in_range(timestamp, start_time, end_time):
         return False
     return (
-        parsed_line.get("level") in levels
```

Comment:
question: is there any context behind this behavior change? this will now print messages that don't have a level field, right? (whereas they were previously filtered out)

Reply:
in the current pipeline, all service logs have "debug" `level` added by the forwarder, which makes  me wonder if there is any usefulness of having it as part of the log event at all?
We haven't added "level" field in the new pipeline, therefore updating this filter condition to either accept new pipeline logs (without level field) or filter other logs if they have a different than default levels, which are defined here: https://github.com/Yelp/paasta/blob/9ed832dc727d46b649d3b7d0153e44a0c7e952bc/paasta_tools/cli/cmds/logs.py#L1422